engines/dreamweb/titles.cpp

```diff
@@ -99,6 +101,9 @@ void DreamWebEngine::gettingShot() {
 }
 
 void DreamWebEngine::bibleQuote() {
+	const char *story = "And I heard a great voice out of the temple saying to the seven angels. Go your ways and pour out the vails of the wrath of god upon the earth. Book of revelation. Chapter 16 verse 7.";
```

Comment:
Please split this line, so it doesn't go too far off the screen. you may write like this:

```C++
story = "This is beginning, "
        "this continues "
        "even more."
```